packages/utils/adapters/package.json

```diff
@@ -25,15 +25,19 @@
     "url": "https://github.com/microsoft/fluentui-react-native.git",
     "directory": "packages/utils/adapters"
   },
+  "dependencies": {
+    "@office-iss/react-native-win32": "^0.72.0",
```

Comment:
Wont this make it so that everyone using FURN will be required to install rn-win32, rn-macos and rn-windows?

That doesn't seem right?

Reply:
I Think this is actually something that could be an optional peer dependency? which is not common but @Tido64 had me do it in RNTA so it's a supported thing: https://github.com/microsoft/react-native-test-app/pull/1803/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

Reply:
Let me undo the changes to this package since we're chatting offline about how to properly fix it